_security/access-control/api.md

```diff
@@ -1399,33 +1486,77 @@ GET _plugins/_security/api/ssl/certs
   ]
 }
 ```
-### Reload certificates
-Introduced 1.0
-{: .label .label-purple }
 
-Reloads SSL certificates that are about to expire without restarting the OpenSearch node. 
+### Reload transport certificates
 
-This call assumes that new certificates are in the same location specified by the security configurations in `opensearch.yml`. To keep sensitive certificate reloads secure, this call only allows hot reload with certificates issued by the same issuer and subject DN and SAN with expiry dates after the current certificate.
+Reload transport layer communication certificates. These REST APIs let a super admin (or a user with sufficient permissions to access this API) reload transport layer certificates.
 
-#### Request
+#### Path and HTTP methods
 
 ```json
-PUT _opendistro/_security/api/ssl/transport/reloadcerts
+PUT /_plugins/_security/api/ssl/transport/reloadcerts
 ```
 {% include copy-curl.html %}
 
+##### Example request
+
+```bash
+curl -X PUT "https://your-opensearch-cluster/_plugins/_security/api/ssl/transport/reloadcerts"
+```
+{% include copy-curl.html %}
+
+##### Example response
+
+```json
+{
+  "status": "OK",
+  "message": "updated transport certs"
+}
+```
+
+#### Response fields
+
+| Field   | Data Type | Description                                                                       |
+|:--------|:----------|:----------------------------------------------------------------------------------|
+| status  | string    | Indicates the status of the operation. Possible values: "OK" or an error message. |
+| message | string    | Additional information about the operation.                                       |
+
+
+#### Reload HTTP certificates
+
+Reload HTTP layer communication certificates. These REST APIs let a super admin (or a user with sufficient permissions to access this API) reload HTTP layer certificates.
+
+#### Path and HTTP methods
+
 ```json
-PUT _opendistro/_security/api/ssl/http/reloadcerts
+PUT /_plugins/_security/api/ssl/http/reloadcerts
 ```
 {% include copy-curl.html %}
 
-#### Example response
+
+##### Example request
+
+```
+curl -X PUT "https://your-opensearch-cluster/_plugins/_security/api/ssl/http/reloadcerts"
+```
+{% include copy-curl.html %}
+
+##### Example response
 
 ```json
-{ 
-  "message": "updated http certs" 
+{
+  "status": "OK",
+  "message": "updated http certs"
 }
 ```
+
+#### Response fields
```

Comment:
I think it's fine to leave this table. But the example response is probably self explanatory for this. It can stay or go.

Reply:
I followed the style doc. Let's keep it?